src/ocm/components/clusterWizard/ClusterDetailsForm.tsx

```diff
@@ -50,7 +51,7 @@ const ClusterDetailsForm: React.FC<ClusterDetailsFormProps> = (props) => {
   } = props;
 
   const featureSupportLevels = React.useContext(FeatureSupportLevelContext);
-  const handleSubmit = async (values: ClusterDetailsValues) => {
+  const handleSubmit = async (values: OcmClusterDetailsValues) => {
```

Comment:
Please wrap it inside `React.useCallback`

Reply:
you mean wrap handleSubmit?

Reply:
yes